File: latihan_3_button/latihan5_BtnNavigasi/app/src/main/java/com/example/latihan5_btnnavigasi/MainActivity.java
package com.example.latihan5_btnnavigasi;

import android.view.MenuItem;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

//import com.example.latihan5_btnnavigasi.fragment_home;

import com.google.android.material.bottomnavigation.BottomNavigationView;
import com.google.android.material.navigation.NavigationBarView;


public class MainActivity extends AppCompatActivity {
    BottomNavigationView bottomNavigationView;
    private final BottomNavigationView.OnNavigationItemSelectedListener navigation = new BottomNavigationView.OnNavigationItemSelectedListener() {
        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {
            Fragment f = null;
//            switch (item.getItemId()){
//                case R.id.menu_home:
//                    f = new fragment_home();
//                    break;
//                case R.id.menu_cart:
//                    f = new fragment_cart();
//                    break;
//                case R.id.menu_notification:
//                    f = new fragment_notif();
//                    break;
//                case R.id.menu_person:
//                    f = new fragment_profile();
//                    break;
//            }
            if (item.getItemId() == R.id.menu_home) {
                f = new fragment_home();
            } else if (item.getItemId() == R.id.menu_cart) {
                f = new fragment_cart();
            } else if (item.getItemId() == R.id.menu_notification) {
                f = new fragment_notif();
            } else if (item.getItemId() == R.id.menu_person) {
                f = new fragment_profile();
            }
            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container_view_tag, f).commit();
            return true;
        }
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        bottomNavigationView = findViewById(R.id.menu_btn_navigasi);
        bottomNavigationView.setOnNavigationItemSelectedListener(navigation);
        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container_view_tag, new fragment_home()).commit();
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container_view_tag, new fragment_home()).commit();
        }
    }
}

File: Tugas1_Register/Zip/Program/app/src/main/java/com/example/tugas1_register/MainActivity.kt
package com.example.tugas1_register

import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        val editTextName = findViewById<android.widget.EditText>(R.id.editTextName)
        val  buttonRegister = findViewById<android.widget.Button>(R.id.buttonRegister)
        val tvwelcome = findViewById<android.widget.TextView>(R.id.tvwelcome)
            
            buttonRegister.setOnClickListener {
            val name = editTextName.text.toString()
            tvwelcome.text = "Welcome $name"

        }
    }
}